src/partitioning/block.jl

```diff
@@ -10,17 +10,19 @@ A method for partitioning spatial objects into blocks of given `sides`.
 """
 struct BlockPartition{Dim,T} <: PartitionMethod
   sides::SVector{Dim,T}
+  neighbors::Bool
 end
 
-BlockPartition(sides::NTuple{Dim,T}) where {Dim,T} = BlockPartition{Dim,T}(sides)
-
-BlockPartition(sides::Vararg{T,Dim}) where {Dim,T} = BlockPartition(sides)
+BlockPartition(sides::NTuple{Dim,T}, neighbors = false) where {Dim,T} = BlockPartition{Dim,T}(sides, neighbors)
+BlockPartition(sides::Vararg{T,Dim}; neighbors::Bool = false) where {Dim,T} = BlockPartition(SVector(sides), neighbors)
 
 function partition(object, method::BlockPartition)
   Dim = embeddim(object)
-
   psides = method.sides
   bbox = boundingbox(object)
+  calculate_metadata = method.neighbors
+  
+  println("Neighbors has been set to ", calculate_metadata)
```

Comment:
You don't want to print something to the screen unless it was asked by the user. Please remove the println statement.

Also notice the convention is to name variables without underscores when possible. I wouldn't even create a new variable for that because `method.neighbors` is already clear in a if statement: `if method.neighbors`

Reply:
Fixed!

Reply:
I don't see the fix. Did you push to GitHub aready? Please do so that we can quickly see the changes and provide feedback.

Reply:
I have committed it now!